homeassistant/components/sensor/rfxtrx.py

```diff
@@ -110,9 +111,9 @@ def __str__(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self.data_type:
+        if self.event:
             return self.event.values[self.data_type]
-        return None
+        return STATE_UNKNOWN
```

Comment:
I don't think this change is needed cause state will be changed to `STATE_UNKNOWN` if state is None by `update_ha_state` method which is called in the end of adding the device.